PriceKataCalc/Program.cs

```diff
@@ -1,2 +1,31 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿using System;
+namespace Kata
+{
+    class priceCalc
+    {
+        public static void Main()
+        {
+            product myProduct = new("ball", 20405, 15.1567,0.20);
+            Console.WriteLine($"Product price reported as {myProduct.price} before tax and {myProduct.price+myProduct.calcTax()} after {myProduct.tax*100}% tax. ");
+
+        }
+       
+    }
+    class product{
+        public String name;
+        public int UPC;
+        public double price;
+        public double tax;
+        public product (String name,int UPC,double price, double tax) { 
+            this.name = name; 
+            this.UPC = UPC;
+            this.price = Math.Round(price, 2);
```

Comment:
This is not enough. Whenever you do a calculation the result would be more than 2 decimals right ?

Reply:
you're right i should've used division